collect_app/src/main/java/org/odk/collect/android/fragments/dialogs/RankingWidgetDialog.java

```diff
@@ -102,6 +102,7 @@ public Dialog onCreateDialog(Bundle savedInstanceState) {
     @Override
     public void onSaveInstanceState(Bundle outState) {
         outState.putSerializable(VALUES, (Serializable) rankingListAdapter.getValues());
+        outState.putSerializable(FORM_INDEX, rankingListAdapter.getFormIndex());
```

Comment:
Why don't you just use `formIndex` here? Then you don't need the method you implemented: `getFormIndex()`?

Reply:
@grzesiek2010 Took care of that. Thanks for pointing out 😄

Reply:
Thanks!